CONTRIBUTING.md

```diff
@@ -0,0 +1,126 @@
+# Overview
+
+This documents explains the processes and practices recommended for
+contributing enhancements to the Prometheus charm.
+
+- Generally, before developing enhancements to this charm, you should consider
+  [opening an issue ](https://github.com/canonical/prometheus-operator) explaining
+  your the use case.
+- If you would like to chat with us about your use-cases or proposed
+  implementation, you can reach us at Canonical
+  [Canonical Mattermost public channel](https://chat.charmhub.io/charmhub/channels/charm-dev).
+- It is strongly recommended that prior to engaging in any enhancements
+  to this charm you familiarise your self with the Juju
+- Familiarising yourself with the
+  [Juju Operator SDK](https://juju.is/docs/sdk) and the Python
+  [Operator Framework](https://ops.readthedocs.io/en/latest/)
```

Comment:
You appear to have referred to the SDK docs and ReadTheDocs as pertaining to two seperate projects? I wouldn't link to the ReadTheDocs page here, it's linked in the juju.is/docs/sdk pages a fair bit

Reply:
Thanks. Changed.